src/components/HomeLandingPage/HomeLandingPage.tsx

```diff
@@ -40,24 +43,71 @@ const HomeLandingPage = () => {
   useAllEvents();
 
   // Fetch events from cache
-  const { data } = useQuery(GET_ALL_EVENTS);
+  const { data } = useQuery(FETCH_ALL_EVENTS);
+  const [editEvent] = useMutation(EDIT_EVENT);
+  const [deleteEvent] = useMutation(DELETE_EVENT, {
+    update(cache, { data: { deleteEvent } }) {
+      if (!deleteEvent) {
+        return;
+      }
+      let { events } = cache.readQuery<any>({
+        query: FETCH_ALL_EVENTS,
+      });
+
+      events = events.filter((event) => event.id !== selectedEvent!.id);
```

Comment:
this line has a linter warning with the following suggested fix - just double check if this does the same as what you originally intended
```suggestion
      events = events.filter((event) => event.id !== selectedEvent?.id);
```